pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -130,6 +132,9 @@ func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err // requeue
 	}
 
+	// Annotate with reconcile-unique UUID to trigger MigMigration reconcile
```

Comment:
Perhaps it would be better to describe this as needed for all remote watch reconcile propagation.

Reply:
Are you thinking we should add the UUID update in other places as well?